src/orquestra/sdk/_base/_config.py

```diff
@@ -214,9 +215,42 @@ def _resolve_auto_config(config_name) -> RuntimeConfiguration:
         "uri": uri,
         "token": passport_token,
     }
+
     return runtime_config
 
 
+def _resolve_local_auto_config(config_env: str) -> RuntimeConfiguration:
+    # if someone sets "auto" as CURRENT_CONFIG_ENV variable, we would get into infinite
+    # recursion here.
+    if config_env == AUTO_CONFIG_NAME:
+        raise exceptions.RuntimeConfigError(
+            f"{AUTO_CONFIG_NAME} can not be the value "
+            f"of {CURRENT_CONFIG_ENV} env variable."
+        )
+    try:
+        config = read_config(config_env)
+    except (exceptions.ConfigFileNotFoundError, exceptions.ConfigNameNotFoundError):
+        raise exceptions.RuntimeConfigError(
+            f"Couldn't find the config {config_env} specified in "
+            f"{CURRENT_CONFIG_ENV} env variable."
+        )
+    return config
+
+
+def _resolve_auto_config(config_name) -> RuntimeConfiguration:
+    # On studio we short-circuit to use internal URIs
+    if is_passport_file_available():
+        return _resolve_remote_auto_config(config_name)
+    elif CURRENT_CONFIG_ENV in os.environ:
+        return _resolve_local_auto_config(os.environ[CURRENT_CONFIG_ENV])
+    else:
+        # we are not in the cluster, and default config env was not set. Error out
+        raise exceptions.RuntimeConfigError(
+            f"Config {config_name} only works on the studio or when the "
+            f"{CURRENT_CONFIG_ENV} env variable is set."
+        )
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:**
```suggestion
        raise exceptions.RuntimeConfigError(
            f"Using '{config_name}' as the config name requires that "
            f"you're  using Studio or that the '{CURRENT_CONFIG_ENV}' "
            "environment variable is set.\n"
            "For example, if you want to use a local Ray cluster, "
            f"set `{CURRENT_CONFIG_ENV}=local`."
        )
```